app/services/anonymized_data_download_service.rb

```diff
@@ -0,0 +1,152 @@
+require 'csv'
+
+module AnonymizedData
+  class DistrictData
```

Comment:
These classes can be moved to their own files.

Reply:
Just the service file is sufficient, and hence the module names can be opaquely namespaced like AnonymizedData::DistrictData etc.

Reply:
@kitallis Moved the DistrictData and FacilityData classes out into individual files.